data/schemas/keyboard.jsonschema

```diff
@@ -276,8 +290,12 @@
         "combo": {
             "type": "object",
             "properties": {
+                "enabled": {"type": "boolean"},
                 "count": {"$ref": "qmk.definitions.v1#/unsigned_int"},
-                "term": {"$ref": "qmk.definitions.v1#/unsigned_int"}
+                "term": {"$ref": "qmk.definitions.v1#/unsigned_int"},
+                "extra_short_combos": {"$ref": "qmk.definitions.v1#/unsigned_int"},
```

Comment:
same with this. Also, this feels like it should be "length" with short, extra short, extra long, etc as options.

Reply:
Yeah, "length" seems like more accurate subkey for thrse.